pisek/task_jobs/generator/pisek_v1.py

```diff
@@ -32,9 +32,14 @@ def __init__(self, env: Env, generator: TaskPath, **kwargs) -> None:
 
     def _run(self) -> list[InputInfo]:
         input_infos = []
+        input_names: set[str] = set()
         self._create_inputs_list()
         for i, line in enumerate(self._get_input_lines()):
-            input_infos.append(self._get_input_info_from_line(line, i))
+            input_info = self._get_input_info_from_line(line, i)
```

Comment:
We should `rstrip("\n")` the line here, so we don't get different number of newlines depending on error.

Reply:
Good point. I have added the _rstrip_ to __line_invalid_ because it is lower level.